plugin/src/main/java/com/denizenscript/denizen/scripts/commands/item/MapCommand.java

```diff
@@ -1,56 +1,53 @@
 package com.denizenscript.denizen.scripts.commands.item;
 
-import com.denizenscript.denizen.scripts.containers.core.MapScriptContainer;
-import com.denizenscript.denizen.utilities.maps.*;
-import com.denizenscript.denizencore.objects.core.ColorTag;
-import com.denizenscript.denizencore.utilities.debugging.Debug;
 import com.denizenscript.denizen.objects.LocationTag;
 import com.denizenscript.denizen.objects.WorldTag;
-import com.denizenscript.denizencore.exceptions.InvalidArgumentsException;
+import com.denizenscript.denizen.scripts.containers.core.MapScriptContainer;
+import com.denizenscript.denizen.utilities.maps.*;
 import com.denizenscript.denizencore.exceptions.InvalidArgumentsRuntimeException;
-import com.denizenscript.denizencore.objects.Argument;
+import com.denizenscript.denizencore.objects.core.ColorTag;
 import com.denizenscript.denizencore.objects.core.ElementTag;
 import com.denizenscript.denizencore.objects.core.ScriptTag;
 import com.denizenscript.denizencore.scripts.ScriptEntry;
 import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.denizencore.scripts.commands.generator.*;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
 import org.bukkit.Bukkit;
 import org.bukkit.map.MapRenderer;
 import org.bukkit.map.MapView;
 
-import java.util.List;
-
 public class MapCommand extends AbstractCommand {
 
     public MapCommand() {
         setName("map");
-        setSyntax("map [<#>/new:<world>] (reset:<location>) (scale:<value>) (tracking) (image:<file>) (resize) (script:<script>) (dot:<color>) (radius:<#>) (x:<#>) (y:<#>) (text:<text>)");
-        setRequiredArguments(2, 10);
+        setSyntax("map [<#>/new:<world>] (reset:<location>/reset_to_blank) (scale:<value>) (tracking) (image:<file>) (resize) (script:<script>) (dot:<color>) (radius:<#>) (x:<#>) (y:<#>) (text:<text>) (width:<#>) (height:<#>)");
+        setRequiredArguments(2, 12);

```

Comment:
I think the limit is 14 actually? It's technically possible, albeit extremely weird, to use every single argument at once. I think?